src/property-filter/index.tsx

```diff
@@ -404,6 +405,7 @@ const PropertyFilter = React.forwardRef(
             </div>
           ) : null}
         </div>
+        {filteringConstraintText && <div className={styles.constraint}>{filteringConstraintText}</div>}
```

Comment:
In a regular form field, constraint text is connected to the input using `aria-describedby`. We should probably do the same here.

Reply:
Great call, I completely missed that. Fixed in the latest revision.